includes/UserMailer.php

```diff
@@ -434,114 +412,189 @@ protected static function quotedPrintableCallback( $matches ) {
  *
  */
 class EmailNotification {
```

Comment:
We talked about moving this logic into the Email extension.  Is this a separate ticket?  Does it have a ticket yet?